bin/install/install_db.inc

```diff
@@ -108,25 +108,26 @@ if (!install_create_config()) {
  * This just prints out the required params for this script
  **/
 function usage() {
-    echo "Usage: install_db.inc [options]";
+    echo T_("Usage:") . "\n";
+    echo "install_db.inc [" . T_('Options') . "]";
     echo "\n\t-U, --database-user\t\t\t";
-    echo 'MySQL Admin User';
+    echo T('MySQL Administrative Username');
```

Comment:
Is there a reason for the missing underscores?

Reply:
No! my wireless keyboard strikes again.